### ISSUE TITLE
Expose endpoints in router

### ISSUE THREAD

**Issue by ferndot** [sunscrapers/djoser#290]
It would be useful if djoser could register its endpoints into a `DefaultRouter`. Then, it would be possible to import its endpoints into the Browsable API root.

I would be willing to contribute this feature.

**Comment by abaybek** [comment]
Year. It would be nice feature. 
My solution was:

`class APIV1View(views.APIView):
    permission_classes = (IsAuthenticated,)

    def get(self, request, format=None):
        content = {
            'auth': reverse('root', request=request, format=format),
            # and etc
}`

**Comment by joshua-s** [comment]
@dekoza is this wanted?

**Comment by dekoza** [comment]
This will be a feature of 2.0 release but this will a big and breaking release. Until that I think we should add this feature to 1.x line.

**Comment by zefciu** [comment]
It turns out this was already implemented, but undocumented.

**Comment by joshua-s** [comment]
@zefciu the existing router only includes the user viewset. Other routes for password changing and token/jwt auth are not present.

**Comment by zefciu** [comment]
Yeah, but these are implemented with views, not viewsets. So putting them into a router will require some more serious refactor.
